src/src/AppBundle/Controller/DefaultController.php

```diff
@@ -87,44 +85,59 @@ public function indexAction($slug, $_locale, Request $request)
             $email = $form->getData()['email'];
             $firstName = $form->getData()['first_name'];
             $lastName = $form->getData()['last_name'];
+            $event = $em->getRepository(Event::class)->find($form->getData()['event_choice']);
+
+            $attendee = $em->getRepository(Attendee::class)->findOneBy(array('email' => $email));
+            if ($attendee) {
+                $attendeeRegistratedForEvent = $em->getRepository(Registration::class)->findOneBy(
+                                                        ['attendee'=> $attendee, 'events' => $event]);
+                if ($attendeeRegistratedForEvent) { // attendee already registered for event
+                    // todo: Viki: problem je v tom, ze view si uz vytvorila a teda sa tato chyba vo view neukaze.
+                    $form->get('email')->addError(new FormError('Attendee already registrated for that event.'));
```

Comment:
prosim ta, vsetky tieto texty tahaj z `lang` suborov, 

`$context['lang']['already_registered']`

a podobne, nech sa daju potom menit, oki?

Reply:
Oki, idem to este opravit.